GitCommands/Git/AheadBehindDataProvider.cs

```diff
@@ -64,22 +69,39 @@ public AheadBehindDataProvider(Func<IExecutable> getGitExecutable)
             }
 
             var matches = _aheadBehindRegEx.Matches(result);
-            if (matches.Count < 1 || (matches.Count == 1 && (!matches[0].Groups["branch"].Success ||
-                                                             !(matches[0].Groups["ahead_p"].Success || matches[0].Groups["ahead_u"].Success ||
-                                                               matches[0].Groups["behind_p"].Success || matches[0].Groups["behind_u"].Success))))
-            {
-                return null;
-            }
-
             var aheadBehindForBranchesData = new Dictionary<string, AheadBehindData>();
             foreach (Match match in matches)
             {
+                var branch = match.Groups["branch"].Value;
+                var remoteRef = (match.Groups["remote_p"].Success && !string.IsNullOrEmpty(match.Groups["remote_p"].Value))
+                            ? match.Groups["remote_p"].Value
+                            : match.Groups["remote_u"].Value;
```

Comment:
```suggestion
                var remoteRef = (match.Groups["remote_p"].Success && !string.IsNullOrEmpty(match.Groups["remote_p"].Value))
                                ? match.Groups["remote_p"].Value
                                : match.Groups["remote_u"].Value;
```

Reply:
why one more level?

Reply:
I'd either `?` align it with the start of the condition or else use one indent level.

Reply:
One indent from 'var'